protocol/message_persistence.go

```diff
@@ -951,13 +951,14 @@ func (db sqlitePersistence) LatestPendingContactRequestIDForContact(contactID st
 	return id, nil
 }
 
-func (db sqlitePersistence) LatestContactRequestIDs() (map[string]common.ContactRequestState, error) {
-	res := map[string]common.ContactRequestState{}
+func (db sqlitePersistence) LatestContactRequestIDs() ([]LatestContactRequest, error) {
```

Comment:
The name of the function is outdated because it's returning more than just IDs now. Could it be renamed to `LatestContactRequests`?